scheduler/src/cook/config.clj

```diff
@@ -613,4 +617,8 @@
 
 (defn task-constraints
   []
-  (get-in config [:settings :task-constraints]))
\ No newline at end of file
+  (get-in config [:settings :task-constraints]))
+
+(defn matching-settings
```

Comment:
Does this fit in this file not at the end? (If everyone puts new config entries at the end, you guarantee merge conflicts.) E.g., after estimated-completion-config?

Reply:
don't we just need a newline at the end of the file?

Reply:
THe idea is to reduce merge conflicts. If I add to the end and you add to the end, there's often going to be a merge conflict.